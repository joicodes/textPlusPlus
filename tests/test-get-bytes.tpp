def void start(){
	int x;
	x = getTextBytes("Hello World", 25, 25);

	/* Hello World is < page_limit so should return bytes in Hello World */
	if(x == 11){
		write("Correct Bytes to fit on Page");
	}else{
		write("Incorrect Bytes to fit on Page");
	}
}