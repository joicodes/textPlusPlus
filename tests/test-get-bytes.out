Correct Bytes to fit on Page
 

